bahir-mqtt/mqtt.py

```diff
@@ -0,0 +1,103 @@
+#
```

Comment:
What's the difference between this newly added file and [this existing one](https://github.com/apache/bahir/blob/master/streaming-mqtt/python/mqtt.py) ?